packages/dapp-toolkit/src/modules/wallet-request/wallet-request-sdk.ts

```diff
@@ -86,15 +91,24 @@ export const WalletRequestSdk = (input: WalletRequestSdkInput) => {
       items,
     }: { interactionId?: string; items: WalletInteraction['items'] },
     callbackFns: Partial<CallbackFns> = {},
-  ): ResultAsync<unknown, SdkError> =>
+  ): ResultAsync<
+    WalletInteractionSuccessResponse,
+    SdkError | WalletInteractionFailureResponse
```

Comment:
can we keep it as `SdkError`?

Reply:
No because it'd not be adequate. It can error with `FailureResponse` that's actually the whole point of this regression :)